config/environments/staging.rb

```diff
@@ -51,13 +51,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
-  config.ssl_options = {
-    hsts: { expires: 2.minutes, subdomains: false },
-    redirect: {
-      exclude: ->(request) { request.path.start_with?('/internal') }
-    }
-  }
```

Comment:
Why did you remove this?

Reply:
I didn't realize at the time this works with `X-Forwarded-Proto`. I assumed this will cause redirect loop if we offload on ALB. Good to revert :+1: